hrjobcontract/js/src/job-contract/controllers/contract.js

```diff
@@ -106,6 +106,89 @@ define([
         return promiseFiles;
       }
 
+      /**
```

Comment:
All of this code is copied from `revision-list`, then why cant we reuse revision-list?